src/EnergyPlus/MoistureBalanceEMPDManager.cc

```diff
@@ -249,9 +249,9 @@ void GetMoistureBalanceEMPDInput(EnergyPlusData &state)
 
         auto const &constr = state.dataConstruction->Construct(surf.Construction);
         auto const *mat = dynamic_cast<const MaterialEMPD *>(s_mat->materials(constr.LayerPoint(constr.TotLayers)));
-        assert(mat != nullptr);
+        // assert(mat != nullptr);
 
-        if (mat->mu > 0.0 && surf.Zone > 0) {
+        if (mat && mat->mu > 0.0 && surf.Zone > 0) {
```

Comment:
Instead, let's just check that mat is real before accessing it.  This will properly pass if the inside layer has EMPD stuff. And it will properly result in a warning message if the layer does not have EMPD stuff.  This works happily now.

Reply:
Just curious, why not not the hasEMPD boolean? isn't what it's for?